src/help.coffee

```diff
@@ -56,9 +58,11 @@ helpContents = (name, commands) ->
   """
 
 module.exports = (robot) ->
-  replyInPrivate = process.env.HUBOT_HELP_REPLY_IN_PRIVATE
+  replyInPrivate      = process.env.HUBOT_HELP_REPLY_IN_PRIVATE
+  privateNotifMessage = process.env.HUBOT_HELP_PRIVATE_MSG || 'replied to you in private!'
```

Comment:
most replies are structured `@${author}, ${msg}` from what I've seen on adapters, so it's lowercase to not make those look odd.